superset-frontend/spec/javascripts/components/Menu_spec.jsx

```diff
@@ -18,7 +18,8 @@
  */
 import React from 'react';
 import { shallow, mount } from 'enzyme';
-import { Nav, MenuItem } from 'react-bootstrap';
+import { Nav } from 'react-bootstrap';
+import { Menu as DropdownMenu } from 'src/common/components';
```

Comment:
VERY minot question (non-blocking), but I'm wondering if just letting it be Menu makes sense... when we could find all the `Menu.Item` within the codebase if we ever needed to.

Reply:
The main component is called `Menu` and the names clashed